data/tribes/scripting/help/building_help.lua

```diff
@@ -511,10 +513,22 @@ function building_help_dependencies_production(tribe, building_description)
 
    -- Produced items
    if (building_description.output_ware_types[1] or building_description.output_worker_types[1]) then
-      result = result .. h3(_"Produces:")
+      result = result .. h2(_"Produces:")
```

Comment:
this upscale in the headline breaks the "dependencies section" in a way that outgoing dependencies are now listed under the new Produces section. Furthermore h2 headlines don't have colomns at he end.  
Suggestion would be to leave the dependencies section as it is and move this new information down to the production section.

Reply:
Moved the section below "Production".

In general the problem is that formatting and calculation is inside specific functions, tied together. Having it now below Production is fine, but formatting isn't nice, imho. "Production" is a h2 header, "Produces" is a h3 header but "Ware[s] produced" and "Ware[s] consumed" are also a h3 header. The problem is that the heading levels are defined in different functions: "Ware[s] produced" is part of this function, but "Ware[s] consumed" is part of the function `help_consumed_wares_workers()`.
![add_consumed_wares](https://user-images.githubusercontent.com/6730556/148260096-b434c2a8-961f-4e33-a217-318a4a2c1e31.jpg)

Reply:
So changing "Ware[s] produced" to a h4 header will leave "Ware[s] consumed" as h3 header. And this is not really nice.

Reply:
Hm, I really would go to drop the Produces: headline here, as we already are in Production.   that would fit nice I believe.